tsconfig.json

```diff
@@ -7,6 +7,14 @@
         "lib": [
             "ES2021"
         ],
+        "paths": {
```

Comment:
Exports have changed in these packages. I think if we were able to use a different module resolution behavior they would not be necessary. Added alias paths here and in webpack config.